frontend/src/pages/ArtistReleaseOverview.js

```diff
@@ -0,0 +1,89 @@
+import {Link, useParams} from "react-router-dom";
+import {useEffect, useState} from "react";
+import {getArtistByName} from "../service/musiComApiService";
+import styled from "styled-components/macro";
+import ArtistHeader from "../components/ArtistHeader";
+import ReleaseDescription from "../components/ReleaseDescription";
+
+export default function ArtistReleaseOverview(){
+    const {artistName} = useParams();
+    const {releaseType} = useParams();
+    const [artist, setArtist] = useState();
+
+    useEffect(() => {
+        getArtistByName(artistName)
+            .then(setArtist)
+            .catch((error) => console.error(error))
+    },[artistName]);
+
+    if (!artist){
+        return(
+            <section>
+                Loading
+            </section>
+        )
+    }
+
+    function releaseList () {
+        if(releaseType === "albums"){
+            return (
+                <>
```

Comment:
don't need fragments here

Reply:
alright, shortens the code a little ...

Reply:
I saw now that I get a compiler error when I delete them ...